dms/static/src/js/views/search_panel.js

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2021 Tecnativa - Víctor Martínez
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+odoo.define("dms.SearchPanel", function(require) {
+    "use strict";
+
+    const core = require("web.core");
+    const SearchPanel = require("web.SearchPanel");
+    const _t = core._t;
+    SearchPanel.include({
+        _renderCategory: function() {
+            var res = this._super.apply(this, arguments);
+            if (this.model === "dms.directory") {
+                var str_search = '<span class="o_search_panel_label_title"><b>';
+                var pos_start = res.search(str_search) + str_search.length;
+                var pos_end = pos_start + res.substring(pos_start).search("</b>");
+                res = res.substring(0, pos_start) + _t("Root") + res.substring(pos_end);
+            }
+            return res;
```

Comment:
```suggestion
            let res = this._super.apply(this, arguments);
            if (this.model === "dms.directory") {
                // Replace label on JS because QWeb has not enough context
                let res_dom = $(res);
                res_dom.find('.o_search_panel_label_title').html(_t('<b>Root</b>'));
                res = res_dom.html();
            }
            return res;
```